libs/SmartSync/SmartSync/Classes/Manager/SFSmartSyncCacheManager.h

```diff
@@ -22,6 +22,8 @@
  WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+#import <SalesforceSDKCore/SalesforceSDKConstants.h>
```

Comment:
These imports are required for `SFSDK_DEPRECATED`. It wasn't complaining on the other targets because it's imported in the `PCH` file.